Clients/GUI/Shared/MainLayout.razor

```diff
@@ -1,22 +1,20 @@
 ﻿@using GUI.Modals.Currency
 @inherits LayoutComponentBase
 
-<div class="sidebar">
-    <NavMenu />
-</div>
-
 <div class="main">
+    <div class="header">
+        <Header />
+    </div>
+    <div class="newsbar float-right">
```

Comment:
It's not news, mb sth like "currencies" idk

Reply:
fixed